mqtt/src/test/scala/docs/scaladsl/MqttSourceSpec.scala

```diff
@@ -98,14 +98,33 @@ class MqttSourceSpec
       connectionSettings.toString should include("ssl://localhost:1885")
       connectionSettings.toString should include("auth(username)=Some(mqttUser)")
     }
+
+    "allow MQTT buffering offline support persistence" in {
```

Comment:
This test doesn't really test the functionality. It simply test the `toString` method....

Reply:
Yes, I'll implement an effective test